src/stylesheets/components/_checklist.scss

```diff
@@ -11,7 +10,6 @@
   margin-bottom: 0;
   margin-top: units(1);
 
-  &.usa-checklist__item--checked,
   // Deprecate usa-checklist__item--checked once the js is updated
```

Comment:
Did you check to see that this is not referenced in the js? If it's all good, we can remove this comment, right?

Reply:
I don't see it in the JS but I do see the class is added. Not sure how it works.

Reply:
nm — I misunderstood what was happening here.